components/director/internal/model/app_template.go

```diff
@@ -9,6 +11,12 @@ type ApplicationTemplate struct {
 	AccessLevel      ApplicationTemplateAccessLevel
 }
 
+type ApplicationTemplatePage struct {
+	Data       []*ApplicationTemplate
```

Comment:
Shouldn't `ApplicationCreateInput` be required and used without pointer?

Reply:
Nope, in models we always use pointers for custom types - see `Application` models or `Runtime` (like `RuntimeStatus`)